src/lib/Key.js

```diff
@@ -28,7 +28,7 @@ class Key {
      * @returns {Nimiq.PublicKey}
      */
     derivePublicKey(path) {
-        return Nimiq.PublicKey.derive(this._derivePrivateKey(path));
+        return Nimiq.PublicKey.derive(this.derivePrivateKey(path));
```

Comment:
I needed to make the `derivePrivateKey` method public, as I need to access the raw private key when signing a swap, to store it in SessionStorage for the iframe.